### PR TITLE
feat(cli): add listen flag

### PR DESCRIPTION
**Pull request by albttx**
# Description

This PR add a `-listen` flag to be able to listen on `0.0.0.0` inside a docker container

### CODE REVIEW

File: gno.land/cmd/gnoland/main.go [hunk 1]
```diff
@@ -105,6 +106,13 @@ func (c *gnolandCfg) RegisterFlags(fs *flag.FlagSet) {
 		"localhost:26657",
 		"replacement for '%%REMOTE%%' in genesis",
 	)
+
+	fs.StringVar(
+		&c.rpcListenAddr,
+		"rpc.laddr",
+		"tcp://127.0.0.1:26657",
```
Comment (moul) [inline gno.land/cmd/gnoland/main.go:113]: This string is used 3 times in this file, and should be a constant. This constant should probably be located in a pkg, and not in the main.

File: gno.land/cmd/gnoland/main.go [hunk 2]
```diff
@@ -116,6 +124,10 @@ func exec(c *gnolandCfg) error {
 		cfg.Consensus.CreateEmptyBlocksInterval = 60 * time.Second
 	})
 
+	if c.rpcListenAddr != "tcp://127.0.0.1:26657" {
+		cfg.RPC.ListenAddress = c.rpcListenAddr
+	}
```
Comment (moul) [inline gno.land/cmd/gnoland/main.go:129]: This is not the way to manage this. Instead, you should parse flags directly against the unmarshalled config file.

**Review by moul — CHANGES_REQUESTED** [review]
I'm not in favor of this shortcut because it creates an inconsistent exception, doesn't reuse the existing constants, and isn't linked with the existing configuration structure.

Please, check #731.

**Comment by thehowl** [comment]
Shall we close this?

**Pull request closed without merging**